### PR TITLE
[13.0][MIG] hr_payroll: Move to OCA payroll

### PR DESCRIPTION
**Pull request by victoralmau**
Move to OCA payroll

Please @pedrobaeza can you review it?

@Tecnativa TT32606

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -120,6 +120,8 @@
     'muk_dms_mail': 'dms',
     'muk_dms_thumbnails': 'dms',
     'muk_dms_view': 'dms',
+    # OCA/payroll
+    'hr_payroll': 'payroll',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py:124]:
```suggestion
    'hr_payroll': 'payroll',
    'hr_payroll_account': 'payroll_account',
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
Should it be in renamed_modules, right?

In addition, where are the migration scripts for these modules? They are not needed?

File: odoo/openupgrade/doc/source/modules120-130.rst [hunk 2]
```diff
@@ -175,9 +175,9 @@ missing in the new release are marked with |del|.
 +----------------------------------------------+-------------------------------------------------+
 |hr_org_chart                                  | Nothing to do                                   |
 +----------------------------------------------+-------------------------------------------------+
-| |del| hr_payroll                             |                                                 |
+| |del| hr_payroll                             | Replaced by OCA module                          |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules120-130.rst]: and the other the same

```suggestion
| |del| hr_payroll                             | Replaced by OCA module `payroll`                  |
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
I think this should be put in `renamed_modules` dict instead, right?

**Comment by pedrobaeza** [comment]
Well, if no conflict, the result is the same, and this way, we avoid possible errors if you already have a module with such name in the previous version, but no strong opinion on it anyways.

**Comment by pedrobaeza** [comment]
@MiquelRForgeFlow, can you make us the favor of generating the analysis file?

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
> can you make us the favor of generating the analysis file?

Ok

**Pull request merged**